### PR TITLE
mpv is ready

### PR DESCRIPTION
**Pull request by port19x**
The thing is now ready to be embedded into qt. Its embedded into qt as is, but not our interface, so we could theoretically ship as is, but embedding this into our UI properly would be ideal. Don't forget our target is May first

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+class Test(QMainWindow):
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.container = QWidget(self)
+        self.setCentralWidget(self.container)
+        self.container.setAttribute(Qt.WA_DontCreateNativeAncestors)
+        self.container.setAttribute(Qt.WA_NativeWindow)
+        player = mpv.MPV(wid=str(int(self.container.winId())),
+                vo='x11', # You may not need this
```
Comment (natis1) [inline main.py]: vo='gpu' for sure. definitely don't use x11.

**Comment by dmnightshade** [comment]
[err.log](https://github.com/possum64team/PossumPipe/files/6408731/err.log)
Currently getting this when I try and run your code.

**Comment by dmnightshade** [comment]
Debug code crashing program.

**Review by dmnightshade — CHANGES_REQUESTED** [review]
needs debug code disabled.

**Comment by port19x** [comment]
Disabled logging and changed vo to gpu

**Review by dmnightshade — APPROVED** [review]
Change is good

**Pull request merged**